code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -120,10 +120,6 @@
 	return
 
 /mob/living/carbon/alien/handle_mutations_and_radiation()
-	if(getFireLoss())
-		if((RESIST_HEAT in mutations) || prob(5))
-			adjustFireLoss(-1)

```

Comment:
How exactly do xenomorphs remove burn damage, now? Is there something with weeds? They don't have healing items.